### PR TITLE
Add a series of  posts about migrating from docker-compose to pods

### PR DESCRIPTION
**Pull request by abalage**
Add a series of two blog posts about migrating from docker-compose services to podman pods.

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Review by TomSweeneyRedHat — COMMENTED** [review]
@abalage Thanks for the blog pointers.  A couple of small things.  Can you please change "podman" to "Podman" except in places where it's a command.  If you don't want to change that on your own site, that's fine, but on this site we'd like to have it properly specified at least.

I'd change the date in the file name of one of these entries from '2020-04-04' to 2020-04-05', whichever one you want to show on the top of the list.   Then copy those files to 2020-04-04-new.md and 2020-04-05-new.md respectively.  Then within each of the new md files, change this line:
```
categories: [blogs]
```
to
```
categories: [new]
```

That way the blog will show up on the main page and then also the page that you see after you press the "blogs" button.

**Comment by abalage** [comment]
Thanks @TomSweeneyRedHat 
I have applied the changes to my branch as per your request. I am also going to update my blog posts too.

File: _posts/2020-04-04-convert-docker-compose-to-pods.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Convert docker-compose services to pods with podman
```
Comment (TomSweeneyRedHat) [inline _posts/2020-04-04-convert-docker-compose-to-pods.md]: @abalage Thanks for putting together the "new" blogs too.  However, I'm not seeing the 'podman' to "Podman" changes throughout out.  For instance, this line should be:

```
title: Convert docker-compose services to pods with Podman
```

Comment (abalage) [reply _posts/2020-04-04-convert-docker-compose-to-pods.md]: I was in hurry and did not push my changes properly. Sorry about that. Now it should be fine.

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM, thanks @abalage !

**Comment by abalage** [comment]
> LGTM, thanks @abalage !

Thank you.

**Pull request merged**